CHANGES.md

```diff
@@ -1,3 +1,20 @@
+## edge-19.7.1
+
+* CLI
+  * Added more descriptive output to the `linkerd check` for control plane
+    ReplicaSet readiness
+  * Renamed `config.linkerd.io/debug` annotation to
+    `config.linkerd.io/enable-debug-sidecar`, to match the CLI flag that sets it
```

Comment:
It may be helpful to say "to match the `--enable-debug-sidecar` flag that sets it" incase a user reading this is not aware of that flag already.